v63004/inref/gtm8923.m

```diff
@@ -0,0 +1,115 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;								;
+; Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	;
+; All rights reserved.						;
+;								;
+;	This source code contains the intellectual property	;
+;	of its copyright holder(s), and is made available	;
+;	under a license.  If you do not know the terms of	;
+;	the license, please stop and do not read further.	;
+;								;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;
+gtm8923
+
+	SET encodings="UTF-16"
+	SET encodings="UTF-16 UTF-16BE UTF-16LE"
+
+	FOR I=1:1:$L(encodings," ") DO
+	. SET chset=$P(encodings," ",I)
+	. SET file="gtm8923"_chset_".file"
+	.
+	. ;READ * test
+	. OPEN file:CHSET=chset:10
+	. USE file:CHSET=chset
+	. WRITE "1",!
+	. CLOSE file
+	. OPEN file:CHSET=chset:10
+	. USE file:CHSET=chset
+	. READ *x,!
+	. CLOSE file
+	. USE $P
+	. WRITE "READ * test on "_chset_" file:"
+	. WRITE x,!
```

Comment:
Can you also do a ZWRITE x. That way we will know what undisplayable bytes if any it contains. Same comment applies to multiple places in this file.